src/Spago/Command/Build.purs

```diff
@@ -42,7 +42,7 @@ type BuildOptions =
   , jsonErrors :: Boolean
   }
 
-run :: forall a. BuildOptions -> Spago (BuildEnv a) Unit
+run :: BuildOptions -> Spago (BuildEnv _) Unit
```

Comment:
What's the reason to wildcard this? Won't this produce warnings?

Reply:
Same question in the other modules, like `Docs` below.

Reply:
It will produce warnings, but we should filter them out - I added all the wildcards after chatting with @mikesol (on Discord, [here](https://discord.com/channels/864614189094928394/867149806178664478/1183383103817924698)) about fitting all of our Envs together, and found that using wildcards gets us better inference - I had a few puzzling errors just go away once I swapped them in. (Mike has [a whole blog post about this](https://dev.to/mikesol/metapolymorphic-providers-160p))